Assets/PatchKit Patcher/Scripts/AppData/HashCalculator.cs

```diff
@@ -27,7 +27,7 @@ public static string ComputeHashString(byte[] buffer, int offset, int length)
 
         public static string ComputeStringHash(string str)
         {
-            return string.Concat(new xxHash(Seed).ComputeHash(Encoding.UTF8.GetBytes(str)).Select(b => b.ToString("X2")));
+            return string.Concat(new xxHash(Seed).ComputeHash(Encoding.UTF8.GetBytes(str)).Select(b => b.ToString("X2")).ToArray());
```

Comment:
Why this change? Wouldn't it break existing usages of ComputeStringHash()?

Reply:
It's something to do with the older C# version.
When I tried outputting the result of this hash it logged something along the lines of 
`System.Linq.Enumerable+<CreateSelectIterator>c__Iterator102[System.Byte,System.String]`, until I added the `ToArray()` call.